concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -269,11 +269,11 @@ resources:
       uri: https://github.com/alphagov/paas-compose-scraper
       tag_filter: v0.1.0
 
-  - name: paas-usage-events-collector
+  - name: paas-billing
     type: git
     source:
-      uri: https://github.com/alphagov/paas-usage-events-collector
-      tag_filter: v0.6.0
+      uri: https://github.com/alphagov/paas-billing
+      tag_filter: 0.7.0
```

Comment:
This has dropped the leading `v` from the tag name, which has caused the pipeline to stall because the tags still have the `v` prefix in Github.

Reply:
Whoops. Being fixed in #1200